DNN Platform/Library/Entities/Users/UserController.cs

```diff
@@ -1885,8 +1885,8 @@ public static bool RestoreUser(ref UserInfo user)
                 //Log event
                 EventLogController.Instance.AddLog("Username", user.Username, portalSettings, user.UserID, EventLogController.EventLogType.USER_RESTORED);
 
-                DataCache.ClearPortalCache(portalId, false);
-                DataCache.ClearUserCache(portalId, user.Username);
+                DataCache.ClearPortalUserCountCache(user.PortalID);
+                DataCache.ClearUserCache(user.PortalID, user.Username);

```

Comment:
I don't understand. Why is this diff not showing changes from #2274 on the left?

Reply:
Yeah same I asked with @ohine in [comment](https://github.com/dnnsoftware/Dnn.Platform/pull/2274#issuecomment-418173141) of #2274